conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/RemoteExceptionMapper.java

```diff
@@ -54,64 +54,35 @@
 
     @Override
     public Response toResponseInner(RemoteException exception) {
-        ErrorType errorType = mapErrorType(exception);
-        Response.ResponseBuilder builder = Response.status(errorType.httpErrorCode());
-
-        try {
-            SerializableError error = SerializableError.builder()
-                    .errorName(errorType.name())
-                    .errorCode(errorType.code().toString())
-                    .errorInstanceId(exception.getError().errorInstanceId())
-                    .build();
-            builder.type(MediaType.APPLICATION_JSON);
-            builder.entity(error);
-        } catch (RuntimeException e) {
-            log.warn(
-                    "Unable to translate exception to json",
-                    SafeArg.of("errorInstanceId", exception.getError().errorInstanceId()),
-                    SafeArg.of("errorName", exception.getError().errorName()),
-                    e);
-            // simply write out the exception message
-            builder.type(MediaType.TEXT_PLAIN);
-            builder.entity("Unable to translate exception to json. Refer to the server logs with this errorInstanceId: "
-                    + exception.getError().errorInstanceId());
-        }
-        return builder.build();
-    }
-
-    private ErrorType mapErrorType(RemoteException exception) {
-        Status status = Status.fromStatusCode(exception.getStatus());
-
-        if (status.getStatusCode() == 401) {
-            log.info(
-                    "Encountered a remote unauthorized exception."
-                            + " Mapping to a default unauthorized exception before propagating",
-                    SafeArg.of("errorInstanceId", exception.getError().errorInstanceId()),
-                    SafeArg.of("errorName", exception.getError().errorName()),
-                    SafeArg.of("statusCode", status.getStatusCode()),
-                    exception);
-
-            return ErrorType.UNAUTHORIZED;
-        } else if (status.getStatusCode() == 403) {
+        if (exception.getStatus() == 401 || exception.getStatus() == 403) {
             log.info(
-                    "Encountered a remote permission denied exception."
-                            + " Mapping to a default permission denied exception before propagating",
+                    "Encountered a remote exception",
                     SafeArg.of("errorInstanceId", exception.getError().errorInstanceId()),
                     SafeArg.of("errorName", exception.getError().errorName()),
-                    SafeArg.of("statusCode", status.getStatusCode()),
+                    SafeArg.of("status", exception.getStatus()),
```

Comment:
Any reason you renamed the args? Slight preference to keep them aligned

Reply:
I noticed that we use `status` in other places so we trying to be internally consistent.
```
conjure-java-runtime pkoenig/propagate .$ rg Arg.*status
conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/RemoteExceptionMapper.java
62:                    SafeArg.of("status", exception.getStatus()),
78:                    SafeArg.of("status", exception.getStatus()),

conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/feignimpl/NeverReturnNullDecoder.java
37:        Preconditions.checkNotNull(object, "Unexpected null body", SafeArg.of("status", response.status()));

conjure-java-jaxrs-client/src/test/java/com/palantir/conjure/java/client/jaxrs/feignimpl/NeverReturnNullDecoderTest.java
49:                .containsArgs(SafeArg.of("status", 200));
59:                .containsArgs(SafeArg.of("status", 200));
69:                .containsArgs(SafeArg.of("status", 200));
```

But changing this is just going to create unnecessary pain for log consumers so I'll revert this.